### ISSUE TITLE
Create Layer Data Model (LDM) for Vision Zero Suite

### ISSUE THREAD

**Issue by frankhereford** [cityofaustin/atd-data-tech#10644]
## Reference Material
* https://github.com/frankhereford/discovery_day_q3_22

**Comment by frankhereford** [comment]
This is outside the scope of this issue, but it would be a worthwhile exercise to make sure all of our lookup tables are still in line with the lookable tables used in the CRIS data. This is not handled by the ETL (now, or ever), and it should be done yearly-or-so.

**Comment by patrickm02L** [comment]
Here's an update on the progress made on the LDM project since January.

**Visualized Flow for the Layer Data Model:**

We have successfully visualized the flow for the LDM. By understanding the data requirements for each layer of the application and how they interact with each other, we created a visual representation that accurately reflects the data flow in the system.

**Gathered Stakeholder and DTS Input:**

We collaborated with stakeholders and DTS to ensure that the LDM model meets their needs and requirements. They provided insights into the business logic, data usage patterns, and the interactions between the various layers of the application.

**Resolved Long Trigger Name Issues to Upgrade Graphql Engine:**

We have resolved the long trigger name issues that were causing problems when upgrading the graphql engine. We renamed the triggers and updated the references to them in the codebase.

**Automated DB Permission Maintenance via Prefect Flow:**

We have successfully automated DB permission maintenance via Prefect Flow. The flow automates the granting and revoking of permissions for various users and roles, saving time and reducing the risk of errors.

**Revised VZ Local Dev Environment Script to be Ubuntu Friendly:**

We have revised the VZ local dev environment script to be Ubuntu-friendly. This ensures that developers using Ubuntu as their operating system can set up their development environment easily and without any issues.

**Prepared Environment for Work:**

We have prepared the environment for work by setting up the necessary tools, libraries, and frameworks required for the task at hand.

**Created LDM Schema:**

We have created a well-defined and maintainable LDM schema, which is crucial for the success of the project. The schema defines the structure and relationships between the various layers of the application.

**Computed 'visionzero' Schema Contents and Stored Them:**

We have computed the contents of the 'visionzero' schema and stored them in a safe and secure manner. This ensures that the application has access to the important data contained in the schema.

**Added New Columns from Updates to CRIS Since We First Created the VZDB:**

We have added the new columns from updates to CRIS since we first created the VZDB. This ensures that the application is up to date and able to handle the latest data.

Overall, we have made significant progress on the Layer Data Model project, and we will continue to work on improving and maintaining the LDM schema as the application evolves.
